config/webpack.dev.js

```diff
@@ -10,7 +10,7 @@ const ENV = process.env.ENV || process.env.NODE_ENV || 'development';
 const API_URL = process.env.API_URL || (ENV === 'inmemory' ? 'app/' : 'http://localhost:8080/api/');
 const FORGE_URL = process.env.FORGE_URL || 'http://localhost:8080/forge';
 const FABRIC8_WIT_API_URL = process.env.FABRIC8_WIT_API_URL || 'http://localhost:8080/api/';
-const REALM = process.env.REALM || "fabric8";
+const FABRIC8_REALM = process.env.FABRIC8_REALM || "fabric8";
```

Comment:
@corinnekrych could you add KEYCLOAK keyword to the env variable name ? I think people will quickly identify to which component belongs.